Podfile.lock

```diff
@@ -15,20 +15,17 @@ DEPENDENCIES:
   - Nimble (~> 7.3)
   - PinLayout
   - Quick (~> 1.3)
-  - Tempura (from `.`)
+  - Tempura (~> 6.0)
```

Comment:
[i] Maybe let's have TempuraTesting point to the local Tempura in the Podfile, and check that everything is set up correctly in the podspec

Reply:
✅